pkg/tfgen/docs.go

```diff
@@ -2358,3 +2361,53 @@ var (
 func guessIsHCL(code string) bool {
 	return guessIsHCLPattern.MatchString(code)
 }
+
+func plainDocsParser(docFile *DocFile, pkgName string, g *Generator) ([]byte, error) {
+	// Replace upstream front matter with Pulumi registry's
+	contentStr, err := replaceUpstreamFrontMatter(string(docFile.Content), pkgName)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: translate code blocks
+
+	//TODO: apply default edit rules
+
+	//TODO: reformat text
+
+	//TODO: Light translation / possible eliding for certain headers such as "Arguments Reference"
```

Comment:
Do you want to merge with these TODOs in place? If so they will need an associated issue.

Reply:
Filed https://github.com/pulumi/pulumi-terraform-bridge/issues/2078 to track this work.